api_tests/collection_submissions_actions/views/test_collection_submissions_actions_detail.py

```diff
@@ -0,0 +1,120 @@
+    node.save()
+    return node
+
+
+@pytest.fixture()
+def collection(collection_provider):
+    collection = CollectionFactory(is_public=True)
+    collection.provider = collection_provider
+    collection.save()
+    return collection
+
+
+@pytest.fixture()
+def collection_submission(node, collection):
+    collection_submission = CollectionSubmission(
+        guid=node.guids.first(),
+        collection=collection,
+        creator=node.creator,
+    )
+    collection_submission.save()
+    return collection_submission
+
+
+@pytest.fixture()
+def collection_submission_action(collection_submission):
+    action = collection_submission.actions.create(
+        from_state='in_progress',
+        to_state='unapproved',
+        trigger='submit',
+        creator=collection_submission.creator,
+        comment='test comment'
+    )
+    return action
+
+
+def configure_test_auth(node, user_role):
+    if user_role is UserRoles.UNAUTHENTICATED:
+        return None
+
+    user = AuthUserFactory()
+    if user_role is UserRoles.MODERATOR:
+        node.provider.get_group('moderator').user_set.add(user)
+    elif user_role in UserRoles.contributor_roles():
+        node.add_contributor(user, user_role.get_permissions_string())
+
+    return user.auth
+
+
+@pytest.mark.django_db
+class TestCollectionSubmissionsActionsDetailGETPermissions:
+    @pytest.mark.parametrize('user_role', UserRoles)
+    def test_status_code__200(self, app, node, user_role, collection_submission_action):
+        test_auth = configure_test_auth(node, user_role)
+        resp = app.get(GET_URL.format(collection_submission_action._id), auth=test_auth, expect_errors=True)
+        assert resp.status_code == 200
+
+    @pytest.mark.parametrize('user_role', [UserRoles.UNAUTHENTICATED, UserRoles.NONCONTRIB])
+    def test_private_collection(self, app, node, collection, user_role, collection_submission_action):
+        collection.is_public = False
+        collection.save()
+        test_auth = configure_test_auth(node, user_role)
+        resp = app.get(GET_URL.format(collection_submission_action._id), auth=test_auth, expect_errors=True)
+        assert resp.status_code in (401, 403)
+
+
+@pytest.mark.django_db
+class TestCollectionSubmissionsActionsDetailGETBehavior:
+    def test_return_action(self, app, node, collection_submission, collection_submission_action):
+        resp = app.get(GET_URL.format(collection_submission_action._id), expect_errors=True)
+        assert resp.json['data']['id'] == collection_submission_action._id
+        assert resp.json['data']['attributes']['from_state'] == 'in_progress'
+        assert resp.json['data']['attributes']['to_state'] == 'unapproved'
+        assert resp.json['data']['attributes']['trigger'] == 'submit'
+        assert resp.json['data']['attributes']['comment'] == 'test comment'
+        assert resp.json['data']['relationships']['creator']['data']['id'] == collection_submission.creator._id
```

Comment:
technically this is the same as the `collection_submission_action.creator._id` but maybe make that explicit to avoid confusion of people reading tests but not fixtures